criu/sk-inet.c

```diff
@@ -572,8 +572,7 @@ static int do_dump_one_inet_fd(int lfd, u32 id, const struct fd_parms *p, int fa
 	if (type != SOCK_RAW)
 		ie.ip_opts->raw = NULL;
 
-	if (pb_write_one(img_from_set(glob_imgset, CR_FD_FILES), &fe, PB_FILE))
-		goto err;
+	pb_write_one(img_from_set(glob_imgset, CR_FD_FILES), &fe, PB_FILE);
```

Comment:
do we need to return -1 if pb_write_one failed?

Reply:
We did not until now, but setting err to -1 would be probably a good thing.